ivadomed/utils.py

```diff
@@ -619,6 +620,51 @@ def threshold_predictions(predictions, thr=0.5):
     return thresholded_preds
 
 
+def segment_volume(model_fname, model-metadata_fname, image_fname, roi_fname=None):
+    """Segment volume.
+    :param model_fname: model filename (.pt) to use.
+    :param model-metadata_fname: model metadata filename (.json), contains model training
```

Comment:
TODO: Use the config_file that is moved to the `log_directory` at the end of the training.

Reply:
TODO: generate a config file for the `testing_data`.